### DISCUSSION TITLE
x-transition inside loop not working?

### DISCUSSION THREAD

**Discussion by Javierko** [alpinejs/alpine#1839] [1. Help] [answered]
Hello community,

I would like to ask how I can achieve `x-transition` animation inside of loop? In Alpine v2 it worked using transition on the first element after `template` tag like it's showed on the screenshot.

![Screenshot_829](https://user-images.githubusercontent.com/33197433/126839702-21610006-0e4b-4fba-99fc-ff208d9b08d9.png)

**Comment by richstandbrook** [comment]
Yeah, I've just been trying that too. Anyone know if we can get looped elements transitioning?

**Reply by SimoTod** [reply]
I think x-transition only works with x-show, try adding `x-show="true"`

**Reply by AndiLeni** [reply]
I have the same problem. Adding x-show did not help. 
Were you able to solve this issue?

**Comment by Javierko** [comment, marked as answer]
Sorry for late response.

I figured out that it works when I use boolean variable in `x-show`, to change variable use `setTimeout` function with default time value.
